### PR TITLE
CHE-669 Make terminal easier to find

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
**_1 Upvote**_ Signed-off-by: Vitaliy Guliy vguliy@codenvy.com

### CODE REVIEW

**Comment by vparfonov** [comment]
+1

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java [hunk 1]
```diff
@@ -229,9 +230,9 @@ private void restoreState(final String machineId) {
             public void apply(List<MachineProcessDto> arg) throws OperationException {
                 for (MachineProcessDto machineProcessDto : arg) {
                     final CommandDto commandDto = dtoFactory.createDto(CommandDto.class)
-                                                            .withName(machineProcessDto.getName())
-                                                            .withCommandLine(machineProcessDto.getCommandLine())
-                                                            .withType(machineProcessDto.getType());
+                            .withName(machineProcessDto.getName())
```
Comment (azatsarynnyy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: seems that something wrong with your formatter

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/NewTerminalAction.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ */
+public class NewTerminalAction extends AbstractPerspectiveAction {
+
+    private final ConsolesPanelPresenter consolesPanelPresenter;
+    private final AnalyticsEventLogger eventLogger;
+
+    @Inject
+    public NewTerminalAction(MachineLocalizationConstant locale,
+                             MachineResources machineResources,
+                               ConsolesPanelPresenter consolesPanelPresenter,
```
Comment (azatsarynnyy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/NewTerminalAction.java]: the same as https://github.com/eclipse/che/pull/645#discussion_r55042735

**Comment by azatsarynnyy** [comment]
Is changes in **built-codeEdit-amd.js** file related to this PR?
May be it was mistakenly committed?

**Comment by vitaliy-guliy** [comment]
A little feature has been added to built-codeEdit-amd.js

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/NewTerminalAction.java (hunk 2 above)
Comment (vitaliy-guliy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/NewTerminalAction.java]: Ok

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java (hunk 1 above)
Comment (vitaliy-guliy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: Thanks

**Comment by azatsarynnyy** [comment]
I think that this little feature should be provided by separate commit in order to avoid problems if someone will decide to cherry-pick this commit for example.

**Pull request merged**